src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/GeneViewSideBar.tsx

```diff
@@ -13,14 +15,19 @@ type Props = {
   activeTabName: SidebarTabName | null;
 };
 
-const GeneViewSidebar = (props: Props) => {
-  if (props.activeTabName === SidebarTabName.OVERVIEW) {
-    return <GeneOverview />;
-  } else if (props.activeTabName === SidebarTabName.EXTERNAL_REFERENCES) {
-    return <GeneExternalReferences />;
-  }
+const client = new ApolloClient({
+  uri: '/thoas'
```

Comment:
Ah! This is a good reminder that Thoas does not support cross-origin requests, which it definitely should.

Reply:
Submitted a PR for Thoas to enable support of cross-origin requests: https://github.com/Ensembl/ensembl-thoas/pull/8. We'll have to wait for Kieron to accept it until the feature branch can properly work. The `/thoas` endpoint with the corresponding proxy config in the dev server will then become redundant.

Reply:
I've now removed the proxy config. Thanks.